thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/controller.js

```diff
@@ -238,12 +238,12 @@ export default Ember.Controller.extend({
       // Now filter results accordingly. TODO: build filtering for each type.
       for (let filter of filterArr) {
         switch(filter.category) {
-          case 'Applications':
+          case 'Applications': {
             this.send('onSearchModeChange', 'Applications');
             let task = this.get('searchByApplicationName');
             let taskInstance = task.perform(filter.filter);
             return taskInstance;
-            break;
+          }
```

Comment:
@newsummit do we need a default case here?

Reply:
Not yet - this is placeholder logic for how to handle the incoming array of filters (still TBD what the default case will be)